src/lib/ngsi/AttributeList.h

```diff
@@ -54,6 +54,16 @@ typedef struct AttributeList
                      const std::string&  indent,
```

Comment:
PR #117 will be a long issue to implement... maybe we should include the CHANGES_NEXT_RELEASE entry in this first PR, for not forgetting in the future.